privileges/privileges.go

```diff
@@ -12,6 +12,8 @@ const (
 	ManufacturerDistributeDevice Privilege = 2
 	// ManufacturerFactoryReset provides access to factory resetting a device.
 	ManufacturerFactoryReset Privilege = 3
+	// ManufacturerDeviceLastSeen indicates when the device last transmitted data. Precision is reduced to the start of 3 hour bucket.
```

Comment:
Maybe "provides access to coarse-grained timestamps indicating when manufacturer's devices last transmitted data"

I would probably not fix the "3 hour" thing here. We will forget to update.